ocaml/xapi-idl/v6/v6_interface.ml

```diff
@@ -78,7 +78,8 @@ type errors =
       (** Thrown by license_check when expiry date matches or precedes current
           date *)
   | License_processing_error  (** License could not be processed *)
-  | License_checkout_error of string  (** License could not be checked out *)
+  | License_checkout_error of string * string
```

Comment:
Since this is a breaking change anyway you could also completely rename the variant to something else?
(Then you could also keep the old one for backwards compatibility if needed)